src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -101,24 +113,38 @@ final class OxfordNanoporeMeasurement {
     }
 
     private void assessState() throws IllegalStateException {
-        // Condition one: Don't allow Fast5 pass and fail folder are empty
-        assessFast5Content()
-        // Condition two: Don't allow Fastq pass and fail folder are empty
-        assessFastQContent()
+        boolean isValid = false
+        // We need to ensure that fastq and fast5 information is provided if guppy basecaller was used
+        if (areFast5FoldersInMeasurement() && areFastQFoldersInMeasurement()) {
+            isValid = true
+        }
+        //// We need to ensure that pod5_skip and fast5_skip information is provided if dorado basecaller was used
+        if (arePod5FoldersInMeasurement()) {
+            isValid = true
+        }
+        if (isValid == false) {
+            throw new IllegalStateException("No valid data is contained in measurement")
+        }
     }
 
-    private void assessFast5Content() throws IllegalStateException {
-        if (folders["fast5pass"].getChildren().isEmpty() && folders["fast5fail"].getChildren()
-            .isEmpty()) {
-            throw new IllegalStateException("The fast5 pass folder and fail folder are empty.")
-        }
+    // Condition one: Don't allow empty Fast5 pass and fail folder
+    private boolean areFast5FoldersInMeasurement() {
+        return isDataFolderInMeasurement("fast5pass") || isDataFolderInMeasurement("fast5fail")
+    }
+    // Condition two: Don't allow empty Fastq pass and fail folder
+    private boolean areFastQFoldersInMeasurement() {
+        return isDataFolderInMeasurement("fastqpass") || isDataFolderInMeasurement("fastqfail")
+    }
+    // Condition three: Don't allow empty Pod5 skip and fast5 skip folder
```

Comment:
all others may be empty?

Reply:
Yes the new data model foresees the following minimal and complete models 
**Minimal Model:** 
![Bildschirmfoto 2023-11-03 um 15 22 55](https://github.com/qbicsoftware/data-model-lib/assets/29627977/7066b8cb-e8b8-4e33-9696-025c9c00c0ca)

**Complete Model:**
![Bildschirmfoto 2023-11-03 um 15 22 59](https://github.com/qbicsoftware/data-model-lib/assets/29627977/73f8f642-71c7-4aad-ada4-67572e36601c)